diff.go

```diff
@@ -11,21 +11,81 @@ import (
 )
 
 // Diff prints a diff between two values to stdout.
+// @group Diff
+//
+// Example: print diff
+//
+//	a := map[string]int{"a": 1}
+//	b := map[string]int{"a": 2}
+//	godump.Diff(a, b)
+//	// <#diff // path:line
+//	// - #map[string]int {
+//	// -   a => 1 #int
+//	// - }
+//	// + #map[string]int {
+//	// +   a => 2 #int
+//	// + }
```

Comment:
I'm unsure whether this change is related to the comment I made earlier (in another PR), when I asked to use Go testable example.

I'm fine with the current PR. I understand what it brings.

But my point was to use Go testable example such as this.


```go
func ExampleDiff() {
        a := map[string]int{"a": 1} 
         b := map[string]int{"a": 2} 
         godump.Diff(a, b) 

         // Output:
         // <#diff // path:line 
         // - #map[string]int { 
         // -   a => 1 #int 
         // - } 
         // + #map[string]int { 
         // +   a => 2 #int 
         // + }
}
```

This special method is used to display examples in the go doc, and leads to have examples that are tested, instead of having a comment that may contain error.